subsys/bluetooth/host_extensions/hci_types_host_extensions.h

```diff
@@ -39,6 +39,14 @@ struct bt_hci_cp_set_power_control_request_param {
 	uint8_t apr_margin;
 } __packed;
 
+#define BT_HCI_OP_VS_SET_ROLE_PRIORITY	BT_OP(BT_OGF_VS, 0x011D)
```

Comment:
Can we have to redefine the commands here or can we simply include the headers from nrfxlib?

Reply:
I think that depends a bit if this file is supposed to be controller-agnostic or not.
Maybe @jori-nordic or @alwa-nordic have an opinion on this?

Reply:
We can start with SDC-specific first.

Reply:
updated.
I guess we could remove the hci_types_host_extensions.h now (I could do that in a separate PR, if you wish).